Readme.md

```diff
@@ -90,6 +90,13 @@ data  = torch.zeros(2, 5, 2)
 output, new_state = layer.forward(data, state) 
 ```
 
+### Example on using the library: Low-level spike raster plots
```

Comment:
I think it might be better not to add this to the Readme in this  form. It is already rather long and the spike rasterplot is not that informative as is.

Reply:
Sounds reasonable. How about moving it to the docs instead?

Reply:
I removed it from the main docs and moved it to the tutorial. We still have the spiking notebook to refer to as well.